tests/app/test_jinja_filters.py

```diff
@@ -57,25 +58,55 @@ def test_get_formatted_currency_with_no_value():
     assert get_formatted_currency("") == ""
 
 
+@pytest.mark.usefixtures("gb_locale")
+@pytest.mark.parametrize(
+    "value, currency, schema_limits, formatted_currency",
+    (
+        (2, "GBP", 6, "£2.00"),
+        (Decimal("2.14564"), "GBP", 6, "£2.14564"),
+        (Decimal("2.1"), "GBP", 1, "£2.1"),
+        (Decimal("2.1"), "GBP", None, "£2.10"),
+        (Decimal("3000"), "JPY", 0, "JP¥3,000"),
+        (Decimal("1.1"), "GBP", 6, "£1.10"),
+        (Decimal("3000.445"), "GBP", 6, "£3,000.445"),
+        (Decimal("3000.44545"), "GBP", None, "£3,000.44545"),
+        ("", "", "", ""),
+    ),
+)
+def test_get_custom_currency(value, currency, schema_limits, formatted_currency, app):
+    with app.app_context():
+        assert (
+            get_formatted_currency(
+                value=value, currency=currency, schema_limits=schema_limits
+            )
+            == formatted_currency
+        )
+
+
 @pytest.mark.usefixtures("gb_locale")
 @pytest.mark.parametrize(
     "number, formatted_number",
     (
         (123, "123"),
         ("123.4", "123.4"),
-        ("123.40", "123.4"),
+        (1, "1"),
+        (1.4, "1.4"),
+        ("123.40", "123.40"),
+        ("123.45678", "123.45678"),
+        ("2344.6533", "2,344.6533"),
         ("1000", "1,000"),
         ("10000", "10,000"),
         ("100000000", "100,000,000"),
         (0, "0"),
-        (0.00, "0"),
+        (Decimal("0.00"), "0.00"),
         ("", ""),
         (None, ""),
         (Undefined(), ""),
     ),
 )
-def test_format_number(number, formatted_number):
-    assert format_number(number) == formatted_number
+def test_format_number(number, formatted_number, app):
```

Comment:
Test jinja filters and placeholder transforms have the same code, we should share the `parametrize` argument between the two instead of duplicating it, as it insures they both test the same scenarios.

Reply:
Sharing the parametrize now 👍